bundling/README.md

```diff
@@ -23,25 +23,22 @@ npm install --save dmn-js
 Use it in your application
 
 ```javascript
-var DmnViewer = require('dmn-js');
-
-var dmnJS = new DmnViewer({
+const dmnJS = new DmnViewer({
   container: '#canvas'
 });
 
-dmnJS.importXML(pizzaDiagram, function(err) {
+try {
+  const { warnings } = await dmnJS.importXML(xml);
 
-  if (!err) {
-    console.log('success!');
+  console.log(`Successful import with ${warnings.length} warnings`);
```

Comment:
I know it's just an example, but maybe there is no need to print warnings if there aren't any. Also, in the case of warnings, I would also print them, just like we do with errors.

Reply:
:+1: 

Done.